test_unstructured/partition/docx/test_doc.py

```diff
@@ -1,214 +1,179 @@
-import os
-from tempfile import SpooledTemporaryFile
+"""Test suite for `unstructured.partition.doc` module."""
+
+from __future__ import annotations
+
+import pathlib
+import tempfile
 
-import docx
 import pytest
 from pytest_mock import MockFixture
 
-from test_unstructured.unit_utils import assert_round_trips_through_JSON, example_doc_path
-from unstructured.chunking.title import chunk_by_title
+from test_unstructured.unit_utils import (
+    CaptureFixture,
+    assert_round_trips_through_JSON,
+    example_doc_path,
+)
+from unstructured.chunking.basic import chunk_elements
 from unstructured.documents.elements import (
     Address,
+    CompositeElement,
+    Element,
     ListItem,
     NarrativeText,
     Table,
+    TableChunk,
     Text,
     Title,
 )
-from unstructured.partition.common import convert_office_doc
 from unstructured.partition.doc import partition_doc
 from unstructured.partition.docx import partition_docx
 
 
-def test_partition_doc_matches_partition_docx(mock_document, expected_elements, tmpdir):
-    docx_filename = os.path.join(tmpdir.dirname, "mock_document.docx")
-    doc_filename = os.path.join(tmpdir.dirname, "mock_document.doc")
-    mock_document.save(docx_filename)
-    convert_office_doc(docx_filename, tmpdir.dirname, "doc")
```

Comment:
These four lines creating the same DOCX file from scratch and then convert it to `.doc` for the test. This code is repeated on almost every test and is not the behavior under test. Replace these documents, which are the same on every test run, with `simple.doc` and `simple.docx` stored in `example-docs/`, then use them directly rather than remake them each time.

This is where the 50% time savings comes from, plus it's a lot easier to read.